### PR TITLE
Update zh.js FINAL VERSION (reviewed by WpyGG) (v3.8.1)

### PR DESCRIPTION
**Pull request by UnityEthan**
This is based on the last change i made, but i added more translations, and improved some of the existing ones. No Google translation is involved. DM Stonkman#0948 on discord for details

### CODE REVIEW

File: zh.js [hunk 1]
```diff
@@ -1,747 +1,747 @@
...
+server.config.fixMov Fixed Movement
+server.config.slidSpd Slide Speed
+server.config.healthMlt 血量倍率 
+server.config.impulseMlt 武器后坐力 
+server.config.nameTeam1 队伍 1 名字 
+server.config.nameTeam2 队伍 2 名字 
+server.config.nameTeam3 队伍 3 名字
+server.config.nameTeam4 队伍 4 名字
+server.config.nameTeam5 队伍 5 名字
+server.config.t1Dmg 队伍 1 伤害
```
Comment (mty22) [inline zh.js:617]: just need to fix whitespace on end of line

Comment (UnityEthan) [reply zh.js:617]: umm idk what spaces do u mean but ill look into it

Comment (UnityEthan) [reply zh.js:617]: oh, i see what you mean. i just cant edit anymore

**Comment by mty22** [comment]
still work in progress

**Comment by WpyGG4125** [comment]
@mty22 what do I do if I have completed the last part of the translation and merged both files?

**Comment by mty22** [comment]
I've made comments in Discord, please have a look:

https://discord.com/channels/448194623580667916/831383180086607922/834227613236461588
https://discord.com/channels/448194623580667916/831383180086607922/834409630837768222

File: zh.js [hunk 2]
```diff
@@ -1,747 +1,747 @@
...
+windows.ranked.header 排位赛
+windows.ranked.login 登录以进行排位赛!
+windows.ranked.level 您至少需要 <span color="black"> {0} 级</span> 以进行排位赛! 当前您的等级是 <span color="black"> {1} 级</span>.
+windows.ranked.mode.r1v1 单排
+windows.ranked.mode.r2v2 双排
+windows.ranked.mode.r4v4 四排
+windows.chall.header 挑战
+windows.chall.login 登录以完成挑战!
+thumbnail.upload.limit 上传失败. 最大缩略图限制为｛0｝KB。
+image.upload.limit 上传失败，最大图片限制为｛0｝KB。
```
Comment (mty22) [inline zh.js:571]: Just noticed that you've used: `。` here, and other parts of the language file has `.`

Can you please update these?

File: zh.js [hunk 3]
```diff
@@ -1,747 +1,747 @@
...
+account.banned.hack 此账号已因为使用外挂被禁赛。
+account.banned.appeal 如果您认为您的账号被误封，请联系：
+account.delete 如果您的账号删除申请在14天內没有被撤销，所有关于此账号的信息将会被删除。
+account.delete.request 您确定您要申请账号删除吗？
+account.delete.cancel 您确定您要撤销账号删除申请吗？
+spin.free.available 免费KR已准备好！
+spin.free.claim  此转盘仅可在每小时完成一局比赛后获得。
+spin.free.disable 免费KR弹窗可以在设置中禁用。
+mod.reset.0 该设置将会在玩家重生后生效。
+mod.reset.1 地图将不会被重设。
```
Comment (mty22) [inline zh.js:720]: Same case here as with my previous comment about: `。`

File: zh.js [hunk 4]
```diff
@@ -1,747 +1,747 @@
...
+stats.nukes 使用核弹数
+stats.melee 近战武器击杀数
+stats.fistkills 空手击杀数
+stats.thrownkills 投掷近战武器击杀数
+stats.headshots 暴击数
+stats.wallbangs 穿墙击杀数
+stats.sprays 涂鸦次数
+windows.host.default.settings 保持玩家计分功能仅可在使用默认设置时被打开
+windows.host.default.commands 房主指令（比如/god, /smite, /noclip, 等等）将会在此模式中被禁用。
+windows.host.rewards 打开KR奖励功能后，玩家可以在地图内获得KR奖励, 但是自定义设置将会被禁用。
```
Comment (mty22) [inline zh.js:732]: Same case here as with my previous comment about: `。`

File: zh.js [hunk 5]
```diff
@@ -1,747 +1,747 @@
...
+windows.store.wheels.sub 试试这些转盘，您将会有机会获得全新皮肤！
+windows.store.currency  *美元和您所在地区货币的汇率将会由支付服务提供商或您使用的银行决定
+windows.store.creator 在购买KR时，请考虑使用创作者专用码以支持我们的视频/地图/模组创作者。
+windows.chall.customs 您不可在自定义房间中完成挑战。
```
Comment (mty22) [inline zh.js:736]: Same case here as with my previous comment about: `。`

**Review by mty22 — CHANGES_REQUESTED** [review]
Just a few minor changes.

Also @WpyGG4125 can you please confirm that you've reviewed? :)

**Comment by WpyGG4125** [comment]
Yes I confirm that I have reviewed and most parts are correct 👍

There are just some little errors:
dev.header is missing, I would suggest to name it 开发者
clans.create is having an excessive 工 (IMO 公会 could be better than 战队 but it's fine)
and I would suggest changing *Premium* to 会员 instead of leaving it as English

Besides these, the file is perfect and ready to go

@mty22 @UnityEthan

**Comment by mty22** [comment]
@UnityEthan can you update the pull request as per @WpyGG4125 's suggestions? We're good to merge after that.

**Comment by mty22** [comment]
@UnityEthan please tell me the work breakdowns between all users who contributed.

File: zh.js [hunk 6]
```diff
@@ -1,747 +1,747 @@
...
+settings.viewmodel.roundArms 圆形手臂
+settings.viewmodel.show.melee 显示近战武器
+settings.viewmodel.show.ads 开镜时隐藏武器
+settings.viewmodel.toggleAim 松手时保持开镜
+settings.viewmodel.show.hands 显示手部
+settings.viewmodel.leftHand 左手持枪
+settings.viewmodel.aimAnim 武器开镜动画
+settings.viewmodel.rot 武器位置旋转
+settings.viewmodel.xOff 武器X轴坐标 (左/右）
+settings.viewmodel.yOff 武器Y轴坐标（上/下）
```
Comment (mty22) [inline zh.js:305]: `(` and `（` are different, are you able to fix this up please?

Ideally go with `(` and `)` to help reduce whitespace

Comment (mty22) [reply zh.js:305]: can you fix this one through out the whole file? find and replace should do the trick!

**Review by mty22 — CHANGES_REQUESTED** [review]

**Comment by UnityEthan** [comment]
**EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k:**
Credits: 1) Stonkman#0948, ign: GV_EthanW, contributed to 80% of the translation
              2) WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
              3) Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation

**Comment by mty22** [comment]
> **EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k:**
> Credits: 1) Stonkman#948, ign: GV_EthanW, contributed to 80% of the translation
> 2) WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
> 3) Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation

this is the simplified chinese translation, not traditional.

**Comment by mty22** [comment]
need final review since last changes: https://github.com/mty22/krunker.io-translations/pull/104/files/6b14f344502acef23a7a54d1f3fcc4c31c8414a7..bc70591e7d1f6914fde719cb64649c1f03799ff4

reviewer can be anyone except UnityEthan.

also @UnityEthan please provide work breakdowns for **Simplified Chinese** update, along with discord users + ign's.

**Comment by mty22** [comment]
need final review since last changes: https://github.com/mty22/krunker.io-translations/pull/104/files/6b14f344502acef23a7a54d1f3fcc4c31c8414a7..bc70591e7d1f6914fde719cb64649c1f03799ff4

reviewer can be anyone except UnityEthan.

also @UnityEthan please provide work breakdowns for **Simplified Chinese** update, along with discord users + ign's.

File: zh.js [hunk 7]
```diff
@@ -475,13 +475,13 @@ windows.host.rehost 请重新开始游戏
...
-windows.publish.map.header 发布或更新游戏
+windows.maps.header 自定义地图
+windows.maps.none 没有找到该地图!
+windows.maps.name 地图名称
+windows.maps.editor 地图编辑器
+windows.maps.publish 发布地图
+windows.maps.community 地图开发者社区
+windows.publish.map.header 发布或更新地图
 windows.publish.map.thumbnail 没有缩略图
 windows.publish.map.paste 粘贴游戏数据
```
Comment (TridentYT) [inline zh.js:486]: 粘贴地图数据

**Comment by UnityEthan** [comment]
EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k:
Credits: 1) Stonkman#948, ign: GV_EthanW, contributed to 80% of the translation
2) WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
3) Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation
**NOTE: THE PERCENTAGES SOLELY REPRESENT THE AMOUNT OF WORK BEING PUT INTO THIS PARTICULAR TRANSLATION, THEY CAN SORT TRADITIONAL OUT THEMSELVES**

**Comment by mty22** [comment]
> EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k

@UnityEthan you're clearly not understanding something here.

This is the Simplified Chinese translation, **not** the Traditional Chinese translation.

The question I've asked the workload breakdown for the Simplified Chinese translation.

**Comment by WpyGG4125** [comment]
> > EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k
> 
> @UnityEthan you're clearly not understanding something here.
> 
> This is the Simplified Chinese translation, **not** the Traditional Chinese translation.
> 
> The question I've asked the workload breakdown for the Simplified Chinese translation.

@mty22 We had some discussions about the division of rewards so we decided to change the rate depending on the reward amount of traditional chinese, if tr chinese offers less rewards then the contribution rate might change

**Comment by mty22** [comment]
> > > EVERYTHING BELOW IS ASSUMING THAT THE REWARD ON TRADITIONAL IS 50k
> > 
> > 
> > @UnityEthan you're clearly not understanding something here.
> > This is the Simplified Chinese translation, **not** the Traditional Chinese translation.
> > The question I've asked the workload breakdown for the Simplified Chinese translation.
> 
> @mty22 We had some discussions about the division of rewards so we decided to change the rate depending on the reward amount of traditional chinese, if tr chinese offers less rewards then the contribution rate might change

In context, Traditional Chinese is a "new language" to Krunker, where as Simplified Chinese is not.

The KR payout for this pull request has zero relation to Traditional Chinese translation as that's completely different.

The KR rewards are set per pull request as I've [already noted here](https://github.com/mty22/krunker.io-translations/blob/main/kr_payment_guide.txt). This won't change unless there's some extremely rare circumstance which dictates once-off change (that of which would be assessed on a case-by-case basis).

Please, stop mixing them together. Keep it simple. This is a public forum, and rules must be adhered to.

Again, please let me know the work breakdowns **_specifically_** for Simplified Chinese.

**Comment by UnityEthan** [comment]
Contributors: 1) Stonkman#094, 90%, ign: GV_EthanW
                      2) WpyGG#4125, 10%, ign: WpyGG

**Comment by WpyGG4125** [comment]
> Contributors: 1) Stonkman#94, 90%, ign: GV_EthanW
> 2) WpyGG#4125, 10%, ign: WpyGG

I have 15% contribution 😬

**Comment by Awesome-Boy-001** [comment]
> Contributors: 1) Stonkman#94, 90%, ign: GV_EthanW
> 2) WpyGG#4125, 10%, ign: WpyGG

I've also contributed to 10% of the translation

**Comment by WpyGG4125** [comment]
Let's just stick to the original division
1) Stonkman#948, ign: GV_EthanW, contributed to 80% of the translation
2) WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
3) Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation
@UnityEthan @mty22 @Awesome-Boy-001

File: zh.js [hunk 8]
```diff
@@ -1,747 +1,747 @@
...
+settings.viewmodel.roundArms 圆形手臂
+settings.viewmodel.show.melee 显示近战武器
+settings.viewmodel.show.ads 开镜时隐藏武器
+settings.viewmodel.toggleAim 松手时保持开镜
+settings.viewmodel.show.hands 显示手部
+settings.viewmodel.leftHand 左手持枪
+settings.viewmodel.aimAnim 武器开镜动画
+settings.viewmodel.rot 武器位置旋转
+settings.viewmodel.xOff 武器X轴坐标(左/右)
+settings.viewmodel.yOff 武器Y轴坐标(上/下）
```
Comment (mty22) [inline zh.js:305]: gotta fix this

File: zh.js [hunk 9]
```diff
@@ -1,747 +1,747 @@
...
+settings.viewmodel.zOff 武器Z轴坐标（前/后）
```
Comment (mty22) [inline zh.js:306]: gotta fix this for consistency sake

File: zh.js [hunk 10]
```diff
@@ -1,747 +1,747 @@
...
+windows.mods.load 加载模组
+windows.mods.none 没有找到任何模组
+windows.mods.header 模组管理器
+windows.mods.guide 模组指南
+windows.mods.list 模组列表
+windows.mods.load.header 加载模组
+windows.mods.publish 发布模组
+windows.mods.viewer 观看者
+windows.account.header 帐号
+windows.account.recover 联系 <a>recovery@yendis.ch</a> 找回账号密码
```
Comment (mty22) [inline zh.js:425]: https://github.com/mty22/krunker.io-translations/blob/main/en.js#L425

need to check what is set across all files

**Comment by mty22** [comment]
> Let's just stick to the original division
> 
> 1. Stonkman#948, ign: GV_EthanW, contributed to 80% of the translation
> 2. WpyGG#4125, ign: WpyGG, contributed to 10% of the translation
> 3. Awesome_Boy#7555, ign: Awesome_Boy, contributed to 10% of the translation


we will go with this

**Comment by mty22** [comment]
OK, merge conflict fixed.

**Comment by mty22** [comment]
KB roles sorted, KR queued.

**Pull request merged**